model_compression_toolkit/quantizers_infrastructure/trainable_infrastructure/keras/base_keras_quantizer.py

```diff
@@ -37,6 +37,7 @@ def __init__(self,
                 quantization_config: quantizer config class contains all the information about a quantizer configuration.
             """
             super().__init__(quantization_config)
+            self.quantizer_parameters = {}
```

Comment:
It can be redefined or just add params to it without redefining.

Reply:
So why do you need it here if you're only initializing it in the inherit class?

Reply:
because I use it in this class in the new function get_trainable_variables, and it should exist. Second thought, It is common for keras and pytorch so I should move it to its parent class.